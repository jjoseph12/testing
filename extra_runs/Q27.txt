{
    switch(+ a b)
    {
        case 1:
        {
            x = + z y;
        }
        case 2:
        {
            c = b;
        }
    }
}