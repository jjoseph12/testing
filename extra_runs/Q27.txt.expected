a: int #
b: int #
x, z, y: ? #
c: int #
